Library/ViewModels/ProjectNavigationSelectorViewModel.swift

```diff
@@ -52,16 +52,27 @@ public final class ProjectNavigationSelectorViewModel: ProjectNavigationSelector
   public init() {
     self.animateButtonBottomBorderViewConstraints = self.buttonTappedProperty.signal
 
+    let displayableTabs: (Project) -> [NavigationSection] = { project in
```

Comment:
Ah so this is handling the conditional loading of the overview tab at first. I must have missed some context while I was out but this experience is cool with product/design?

Reply:
Yea it should be, because it really handles all our failing cases for the pledge cta pretty nicely. Which solves the issues raised in this [ticket](https://kickstarter.atlassian.net/browse/NTV-258) as well, which is a nice win.

Essentially the pledge cta is tied only to the overview tab which is always loaded when the vc viewDidLoad gets called.
So any failure/inability for the network or project to have `extendedProjectProperites` (which currently we only get through these new network requests) means we don't have to worry about confusion when the tabs are visible, they just simply are not created until the data is there. As a test can use Charles to fail the network request on this page, then "retry" on the pledge cta and see the tabs appear.
 
I believe product is good with this because they approved this [ticket](https://kickstarter.atlassian.net/browse/NTV-294) for release which is related to the same issue above.